### ISSUE TITLE
Few suggestions for extended documentation.

### ISSUE THREAD

**Issue by harshithjv** [uktrade/stream-zip#25]
Thanks for this great tool! Really awesome one.

I'm able to stream files directly from files stored as blobs on cloud(`Microsoft Azure`) directly to zip file via. streaming http response. For backend server, we're using `Django` web framework and `FileResponse`(child class of `StreamingHttpResponse`) object is used to serve the zip file.

I'm write this issue to provide some suggestions:
- A detailed document, ideally hosted on Read the Docs website.
- Include examples of using this library in conjunction with multiple python libraries and/or frameworks.
- In your main example, elaborate yield statements with examples for multiple yield calls within individual inner function calls:
```py
def unzipped_files():
    modified_at = datetime.now()
    perms = 0o600

    def file_1_data():
        yield b'Some bytes 1'

    def file_2_data():
        yield b'Some bytes 1'
        yield b'Some bytes 2'

    def file_3_data():
        yield b'Some bytes 1'
        yield b'Some bytes 2'
        yield b'Some bytes 3'
        yield b'Some bytes 4'

    def file_4_data():
        for i in range(5):
            yield bytes(f'Some bytes {i}', encoding="utf-8")
```

**Comment by michalc** [comment]
Hi @harshithjv 

Great to hear that this is useful to you, and thanks for the suggestions. To let you know we have made some changes in response:

- We've moved the README contents to https://stream-zip.docs.data.trade.gov.uk/ . It's not on Read the docs, and it could use a bit of polish I suspect, but I think it's a step forward

- I've put your example in what is now the getting started section https://stream-zip.docs.data.trade.gov.uk/getting-started/

On having more examples, I think for now I'll leave it to you. If you (or others reading this) would like to raise a PR against the new documentation at https://github.com/uktrade/stream-zip/tree/main/docs with examples, you're very welcome, but in the meantime, I think I'll close this issue.

Thanks,

Michal
